### ISSUE TITLE
vue devtools is blank

### ISSUE THREAD

**Issue by breekoy** [SimulatedGREG/electron-vue#982]
I can't view vue devtools. It's just a blank page.

Open devtools, click Vue tab.

![Screen Shot 2020-02-10 at 2 51 33 PM](https://user-images.githubusercontent.com/9507381/74127484-c157c380-4c15-11ea-98a7-cb312992a75f.png)

also my terminal returns this errors:

```
┏ Electron -------------------

  2020-02-10 15:23:24.257 Electron[5045:71217] *** WARNING: Textured window <EventDispatchingWindow: 0x7f891c4791d0> is getting an implicitly transparent titlebar. This will break when linking against newer SDKs. Use NSWindow's -titlebarAppearsTransparent=YES instead.

┗ ----------------------------

┏ Electron -------------------

  [5045:0210/152344.653738:ERROR:CONSOLE(1)] "Uncaught SyntaxError: Unexpected token ...", source: chrome-extension://vue-js-devtools/build/devtools.js (1)

┗ ----------------------------
```

  * Node version: v12.15.0
  * NPM version: 6.13.7
  * vue-cli version: 4.2.2
  * electron: 1.7.2
  * Operating System: MacOS 10.15.3

UPDATE: tried to downgrade to node `v10.16.0` and npm `v6.9.0` since these are the versions currently installed in my other machine (linux) which the vue devtools works. Unfortunately, still not working on my mac. 😢

**Comment by breekoy** [comment]
additionally, my terminal returns this error:
```
┏ Electron -------------------

  2020-02-10 15:23:24.257 Electron[5045:71217] *** WARNING: Textured window <EventDispatchingWindow: 0x7f891c4791d0> is getting an implicitly transparent titlebar. This will break when linking against newer SDKs. Use NSWindow's -titlebarAppearsTransparent=YES instead.

┗ ----------------------------

┏ Electron -------------------

  [5045:0210/152344.653738:ERROR:CONSOLE(1)] "Uncaught SyntaxError: Unexpected token ...", source: chrome-extension://vue-js-devtools/build/devtools.js (1)

┗ ----------------------------
```

**Comment by breekoy** [comment]
tried to downgrade to node `v10.16.0` and npm `v6.9.0` since these are the versions currently installed in my other machine (linux) which the vue devtools works. Unfortunately, still not working on my mac. :(

**Comment by breekoy** [comment]
SOLVED! Apparently, this is not an issue with this repo but an issue with the chrome version i'm having. It's too old to work with latest version of vue devtools extension specifically the Chrome 58 that ships with my electron 1.7.2. Also I found out that the last version of chrome where the latest vue devtools is working is chrome 60. So I tried upgrading the electron to v2.0.0 which ships with chrome 61 and it works!
